### PR TITLE
Apply automatic formatting and code cleanup instruments

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
It is necessary to adjust the current codebase to a single formatting standard and also perform some other small cleanups here and there.

### CODE REVIEW

File: app/src/main/java/io/appium/uiautomator2/handler/FindElement.java [hunk 1]
```diff
@@ -140,8 +149,8 @@ public AppiumResponse safeHandle(IHttpRequest request) {
         }
     }
 
-    private Object findElement(By by) throws InvalidSelectorException, ElementNotFoundException,
-            ParserConfigurationException, ClassNotFoundException, UiSelectorSyntaxException,
+    private Object findElement(By by) throws
+            ClassNotFoundException,
```
Comment (vmaxim) [inline app/src/main/java/io/appium/uiautomator2/handler/FindElement.java]: Extra break

File: app/src/main/java/io/appium/uiautomator2/handler/FindElement.java [hunk 2]
```diff
@@ -159,9 +168,9 @@ private Object findElement(By by) throws InvalidSelectorException, ElementNotFou
         throw new UnsupportedOperationException(msg);
     }
 
-    private Object findElement(By by, String contextId) throws InvalidSelectorException,
-            ClassNotFoundException, UiSelectorSyntaxException,
-            UiAutomator2Exception, UiObjectNotFoundException, ElementNotFoundException {
+    private Object findElement(By by, String contextId) throws
+            ClassNotFoundException,
```
Comment (vmaxim) [inline app/src/main/java/io/appium/uiautomator2/handler/FindElement.java]: Extra break.

File: app/src/main/java/io/appium/uiautomator2/model/UiObject2Element.java [hunk 3]
```diff
@@ -97,7 +96,7 @@ public String getStringAttribute(final String attr) throws UiObjectNotFoundExcep
     }
 
     public boolean getBoolAttribute(final String attr)
-            throws UiObjectNotFoundException, NoAttributeFoundException, UiAutomator2Exception {
+            throws UiAutomator2Exception {
```
Comment (vmaxim) [inline app/src/main/java/io/appium/uiautomator2/model/UiObject2Element.java]: Extra break.

**Comment by mykola-mokhnach** [comment]
@vmaxim All fixed

**Comment by mykola-mokhnach** [comment]
@vmaxim Is [this](https://travis-ci.org/appium/appium-uiautomator2-server/jobs/373448555) test unstable?

File: app/src/main/java/io/appium/uiautomator2/handler/FindElement.java [hunk 4]
```diff
@@ -140,18 +140,14 @@ public AppiumResponse safeHandle(IHttpRequest request) {
         } catch (UiSelectorSyntaxException e) {
             Logger.error("Unable to parse UiSelector: ", e);
             return new AppiumResponse(getSessionId(request), WDStatus.UNKNOWN_COMMAND, e);
-        } catch (UiAutomator2Exception e) {
+        } catch (Exception e) {
             Logger.error("Exception while finding element: ", e);
             return new AppiumResponse(getSessionId(request), WDStatus.UNKNOWN_ERROR, e);
-        } catch (UiObjectNotFoundException e) {
```
Comment (vmaxim) [inline app/src/main/java/io/appium/uiautomator2/handler/FindElement.java]: Are you sure that this catch block can be safely deleted?

**Pull request merged**